features/step_definitions/basic_steps.rb

```diff
@@ -31,12 +31,16 @@
 def page_path(path)
   if path == 'Create Article'
     new_article_path
-  elsif path == 'Holger is the best'
-    article_title = Article.find_by(title: path)
-    article_path(article_title)
   elsif path == 'Index'
     root_path
+  elsif (path == 'Holger is the best') || (path == 'Jade loves cookies') # || (add new path here)
```

Comment:
I'm still not convinced that this is a good solution but it is NOT a blocker.

Reply:
Did you read my last response for this one? :)